contracts/StablecoinConverter.sol

```diff
@@ -261,8 +280,9 @@ contract StablecoinConverter is EpochTokenLocker {
         //                    = (executedBuyAmount * buyTokenPrice / sellTokenPrice) * (feeDenominator - 1) / feeDenominator
         //                    in order to minimize rounding errors, the order is switched
         //                    = (executedBuyAmount * buyTokenPrice / feeDenominator) * (feeDenominator - 1) / sellTokenPrice
-        uint256 sellAmount = (uint256(executedBuyAmount).mul(buyTokenPrice) / (feeDenominator - 1))
-            .mul(feeDenominator) / sellTokenPrice;
+        uint128 scaledFeeDenominator = scale * feeDenominator;
+        uint256 sellAmount = (uint256(executedBuyAmount).mul(buyTokenPrice) / (scaledFeeDenominator - 1))
```

Comment:
IN the comments, it says that (feeDenominator - 1) should be the numerator.
But you are using (scaledFeeDenominator - 1) as a denominator. What is true?

Reply:
Interesting... I am not sure what happened here. This is a really good catch! For reference, I will include the comments above and adjust this immediately!

```
        // executedSellAmount = sellAmount * (1 - (1/feeDenominator))
        //                    = sellAmount - sellAmount/feeDenominator
        //                    = (sellAmount * feeDenominator) / feeDenominator - sellAmount/feeDenominator
        //                    = (sellAmount * feeDenominator - sellAmount) / feeDenominator
        //                    = (sellAmount * (feeDenominator - 1)) /feeDenominator
        //                    = (executedBuyAmount * buyTokenPrice / sellTokenPrice) * (feeDenominator - 1) / feeDenominator
        //                    in order to minimize rounding errors, the order is switched
        //                    = (executedBuyAmount * buyTokenPrice / feeDenominator) * (feeDenominator - 1) / sellTokenPrice
```

Reply:
Also, we should be using `SafeMath.div` here.

Reply:
Can we add a test case that would have revealed this bug?

Reply:
This is actually not a bug. it is just that the commented equation here was outdated! Nothing to see here boys.